# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/08
# fix sphire installation path
diff -ru eman2-2.99.47/sphire/CMakeLists.txt eman2-2.99.47_sphire_install_dir/sphire/CMakeLists.txt
--- eman2-2.99.47/sphire/CMakeLists.txt	2023-03-04 20:27:15.000000000 +0100
+++ eman2-2.99.47_sphire_install_dir/sphire/CMakeLists.txt	2023-08-22 17:30:33.383741803 +0200
@@ -1,2 +1,2 @@
 # Install sphire via pip
-INSTALL(CODE "execute_process(COMMAND ${Python3_EXECUTABLE} -m pip install ${CMAKE_CURRENT_SOURCE_DIR} --no-compile)")
+INSTALL(CODE "execute_process(COMMAND ${Python3_EXECUTABLE} -m pip install ${CMAKE_CURRENT_SOURCE_DIR} --no-compile --no-deps  --ignore-installed  --no-index  --no-build-isolation --prefix=${EMAN_PREFIX})")
